runtime/autoload/health.vim

```diff
@@ -9,13 +9,20 @@ function! s:enhance_syntax() abort
   highlight link healthInfo ModeMsg
 
   syntax keyword healthSuccess SUCCESS
-  highlight link healthSuccess Function
+  highlight link healthSuccess ModeMsg
 
   syntax keyword healthSuggestion SUGGESTIONS
   highlight link healthSuggestion String
 
+  syntax match healthHelp "|.\{-}|" contains=healthBar
+  syntax match healthBar  "|" contained conceal
+  highlight link healthHelp Identifier
+
   " We do not care about markdown syntax errors in :CheckHealth output.
   highlight! link markdownError Normal
+
+  " We don't need code blocks.
+  syntax clear markdownCodeBlock
```

Comment:
I was thinking we might want to use these at some point, no? At least the fence, not language-specific codeblocks though.